packages/api/db/migrations/20200712085743_add_emailkey.ts

```diff
@@ -0,0 +1,13 @@
+import Knex from "knex";
+
+export const up = async (knex: Knex) => {
+    return knex.schema.table("users", table => {
+        table.string("emailKey");
+    });
+};
+
+export const down = async (knex: Knex) => {
+    return knex.schema.table("users", table => {
```

Comment:
Braces, you just told me to not put them when I can directly return, but it's ok hehe.